aggregator/src/param.rs

```diff
@@ -21,7 +21,7 @@ impl ConfigParams {
         Self {
             strategy: FpStrategy::Simple,
             degree: 21,
-            num_advice: vec![63],
+            num_advice: vec![100],
```

Comment:
I think we have just induced several cols but here the advice cols increase from 63 -> 100 ?

Reply:
I have reduced it to the minimum possible, that allows the integration tests to run.